java/datastore/src/main/java/com/google/datastore/v1/client/RemoteRpc.java

```diff
@@ -49,6 +51,21 @@
   private final HttpRequestInitializer initializer;
   private final String url;
   private final AtomicInteger rpcCount = new AtomicInteger(0);
+  private static boolean enableE2EChecksum;
+
+  static {
+    InputStream inp = RemoteRpc.class.getClassLoader().getResourceAsStream("config.properties");
+    if (inp == null) {
+      throw new IllegalArgumentException("can't find properties file");
+    }
+    Properties props = new Properties();
+    try {
+      props.load(inp);
+    } catch (IOException e) {
+      throw new IllegalArgumentException("can't load properties file");
+    }
+    enableE2EChecksum = Boolean.parseBoolean(props.getProperty("e2e-checksum"));
```

Comment:
None of our java libraries use config files for configurable options. Most often bootstrap type options are exposed via environment variable (sometimes with a Java system property fallback). Can we switch this to an env variable instead, something like `GOOGLE_CLOUD_DATASTORE_HTTP_ENABLE_E2E_CHECKSUM`?

Reply:
Done